molecule/default/tests/test_dashboard.py

```diff
@@ -4,7 +4,7 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('dashboard_hosts')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('infra1')
```

Comment:
Is there a way to use the `dashboard_hosts` group and use the first element of the array of hosts associated with it? I am wondering if that would make it less coupled specifically to the MNAIO deployment. If we use `infra1` these tests will fail on an `aio` deployment.

Reply:
I presume the command would be the following:
```
testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('dashboard_hosts')[0]
```

Reply:
for infra_host in infra_hosts:
    if infra_host == 'infra1 or infra_host == infra:
        //run test

Reply:
My previous attempt failed because it assigned a scalar value to `testinfra_hosts` rather than an single item list. This works better:
```
testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('dashboard_hosts')[:1]
```

Example session on AIO deploy
```
   ============================= test session starts ==============================
    platform linux2 -- Python 2.7.12, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
    rootdir: /tmp/system_test_workingdir.QWNVfxSm/rpc-openstack-system-tests/molecules/molecule-validate-cinder-deploy/molecule/default, inifile:
    plugins: testinfra-1.7.1, rpc-0.1.0
collected 7 items

    tests/test_default.py ..ssss                                             [ 85%]
```